### ISSUE TITLE
Could your sketch support iPad 12.9 template?

### ISSUE THREAD

**Issue by tuyenbq-0446** [jtholloran/shots#1]

**Comment by kajensen** [comment]
Agreed. This is a great resource, and thank you for creating it @jtholloran. All its missing is iPad support (and maybe tvOS, but mostly iPad) :)

**Comment by MehdiChennoufi** [comment]
Any news about it ?

**Comment by rakithape** [comment]
Appreciate you attention on this. Thanks

**Comment by michaelcohen** [comment]
We'd love that, too!

**Comment by cubissimo** [comment]
I think it is reasonable, since ipad screenshots are mandatory by apple as well

**Comment by doginthehat** [comment]
Yes pleaaaaase

**Comment by newmesiss** [comment]
???

**Comment by alouanemed** [comment]
A lot of people would appreciate it if you do this @jtholloran, as it's obligatory in the app store. and btw add a sponsor badge in your repo. Thanks.

**Comment by jtholloran** [comment]
Are iPad screenshots really required when submitting an app developed for iPhone only? Seems to be some confusion about that. I'm definitely confused since I haven't run into that issue. @alouanemed @cubissimo Can you point to some docs or explain further?

I'd like to add iPad support, but haven't had the time or personal need, since I primarily design iPhone apps — apps that are on the App Store and haven't had any developers come back saying they need iPad shots to submit.

**Comment by alouanemed** [comment]
Yes, I just submitted a new app yesterday and it is required to upload 3 screenshots for 2nd and 3rd gen iPad Pro each.

I spent a day looking for the 2nd generation iPad Pro mockup and generating iPad screenshots.

Thanks!

**Comment by softmarshmallow** [comment]
any updates?

**Comment by Mogikan** [comment]
This guy (https://vk.com/id144849788) added iPad and Watch devices, but it's in Russian. Maybe someone will be capable of merging his skins to this repo https://disk.yandex.ru/d/iZlsOBYqUW8RBg
